### ISSUE TITLE
fail to reproduce the repo

### ISSUE THREAD

**Issue by hariszaf** [aitgon/vtam_benchmark#1]
Hi there. 

i am trying to reproduce your benchmark and i keep getting the following error: 

```bash
10 of 78 steps (13%) done
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2022-11-14T142154.012588.snakemake.log

```
Attached you may find the corresponding log file

[2022-11-14T142154.012588.snakemake.log](https://github.com/aitgon/vtam_benchmark/files/10003606/2022-11-14T142154.012588.snakemake.log.txt)

**Comment by hariszaf** [comment]
I managed to make this work but I have stacked at a next step. 
(the first error was due to lack of one of the requirements, so my bad completely!) 
[2022-11-14T171523.177917.snakemake.log](https://github.com/aitgon/vtam_benchmark/files/10005189/2022-11-14T171523.177917.snakemake.log)
Thanks for your help.

**Comment by aitgon** [comment]
Hi,

We have released a new version 0.1.1 to fix these problems. Everything works within singularity to fix the requirements problems, with a minimal conda installation. If you want to use directly a built singularity image, you can get it in the release package: [https://github.com/aitgon/vtam_benchmark/releases/tag/0.1.1](https://github.com/aitgon/vtam_benchmark/releases/tag/0.1.1)

Thanks for trying it.
